framework/src/st_things/st_things_request_handler.c

```diff
@@ -83,7 +83,7 @@ bool get_query_value_internal(const char *query, const char *key, char **value,
 
 	bool res = false;
 	while (p_ptr != NULL) {
-		if (strncmp(p_ptr, key, key_len) == 0) {
+		if (strncmp(p_ptr, key, sizeof(MAX_KEY_LENGTH)) == 0) {
 			THINGS_LOG_V(THINGS_DEBUG, TAG, "Key(%s) exists in query parameter(%s).", key, query);
```

Comment:
MAX_KEY_LENGTH would be constant,
sizeof also constant. right?
which one do you want to use? I suggest to reomve sizeof